d2bs/kolbot/libs/bots/TeamDiablo.js

```diff
@@ -0,0 +1,462 @@
+/**
+ *    @filename    Teamdiablo.js
+ *    @author      Jaenster
+ *    @desc        Make a true party of a baalrun.
+ */
+
+/*
+ */
+function TeamDiablo() {
+    var diablo = {
+        run: function () {
+            // Init the team, if not already done
+            Team.init();
+            Team.msg.requestData('diablo');
+
+            // Get to throne
+            if (Config.TeamDiablo.Teleport && !Team.communication.diablo.portalReady) {
+                diablo.common.teleport();
+            } else {
+                Team.util.goBo();
+                Town.doChores();
+                Town.goToTown(4)
+            }
+
+            if (!Config.TeamDiablo.Teleport && Team.util.doingBaal() && Config.TeamDiablo.TeleportToThrone) {
+                diablo.util.print('Teleporting to throne');
+                Pather.useWaypoint(Areas.WorldstoneLvl2);
+
+                if (!Pather.moveToExit([Areas.WorldstoneLvl3, Areas.ThroneOfDestruction], true)) {
+                    throw new Error("Failed to move to Throne of Destruction.");
+                }
+
+                Pather.moveTo(Team.data.baalPortalSpot[0],Team.data.baalPortalSpot[1]);
+                Pather.makePortal(true);
+
+                Team.communication.baal.AtWorldstoneLvl3 = true;
+                Team.communication.baal.portalReady = true;
+                Team.msg.sendData('baal'); // send update so everyone know the portal is ready
+            }
+
+            // Run all seals
+            var who;
```

Comment:
'who' does not need to be in a global scope, it is not used outside the context of the for loop itself